server/database/models/index.js

```diff
@@ -39,4 +39,4 @@ Object.keys(db).forEach(modelName => {
 db.sequelize = sequelize;
 db.Sequelize = Sequelize;
 
-export default db;
+module.exports = db;
```

Comment:
Why was this changed back to `module.exports = db`

Reply:
@andela-gike
The model class was `undefined` when I imported it when I used `export default` but it worked when I changed it back to `module.exports`